src/components/ActionDetailsList.tsx

```diff
@@ -304,18 +357,7 @@ function getColumns(intl: InjectedIntl): IRenderableColumn[] {
             // Should be action.negativeEntities.join('').toLowerCase(), but need entity names which requires lookup
             // This lookup should be done ahead of time instead of on every render
             getSortValue: action => '',
-            render: (action, component) => action.requiredEntities.length === 0
-                ? <OF.Icon iconName="Remove" className="cl-icon" data-testid="action-details-empty-required-entities" />
-                : action.requiredEntities.map(entityId => {
-                    const entity = component.props.entities.find(e => e.entityId === entityId)
-                    return (
-                        <div className='ms-ListItem is-selectable ms-ListItem-primaryText' key={entityId} data-testid="action-details-required-entity">
-                            {entity
-                                ? entity.entityName
-                                : `Error - Entity ID: ${entityId}`}
```

Comment:
Can you remind me in which cases we will have an an `entityId` but no `entity`?
Only one I could think of is because we allow deleting an entity even if it's in use.  All the actions and train dialogs that referenced it become invalid and show this error?

Reply:
With the most recent version we try to let users more or less do whatever they want in terms of editing, with warnings and errors when they do so.  Also, if multiple people are editing the same model one could end up a weird state.  Generally trying to detect all these cases and handle them gracefully.